tests/robustness/validate/validate.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 // ValidateAndReturnVisualize returns visualize as porcupine.linearizationInfo used to generate visualization is private.
-func ValidateAndReturnVisualize(t *testing.T, lg *zap.Logger, cfg Config, reports []report.ClientReport) (visualize func(basepath string) error) {
+func ValidateAndReturnVisualize(t *testing.T, lg *zap.Logger, cfg Config, reports []report.ClientReport, withPrevKV bool) (visualize func(basepath string) error) {
```

Comment:
Don't think we need to pass `withPrevKV` here. We can check if WatchRequest has `PrevKV` set.